gbpservice/neutron/services/servicechain/drivers/oneconvergence_intercloud_driver.py

```diff
@@ -13,12 +13,11 @@
 import ast
 
 from neutron.common import log
+from neutron.api.v2 import attributes
 from neutron.openstack.common import jsonutils
 from neutron.openstack.common import log as logging
-from neutron.plugins.common import constants as plugin_constants
 from neutron.plugins.common import constants as pconst
-from neutron.services.loadbalancer.drivers.OneConvergence.device_manager  \
-    import DeviceManager as ServiceManager
+from neutron.services.oc_device_manager.service_manager_client import SvcManagerClientApi
 from oslo.config import cfg
 from neutron import manager
```

Comment:
imports should be in alphabetical order. Please run tox -e pep8 from the repo and fix the reported errors.